packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3064,30 +3061,34 @@ export default class Meeting extends StatelessWebexPlugin {
       this,
       {
         file: 'meeting/index',
-        function: 'setLocalTracks',
+        function: 'sendLocalMediaReadyEvent',
       },
       EVENT_TRIGGERS.MEDIA_READY,
       {
         type: EVENT_TYPES.LOCAL,
         stream: MediaUtil.createMediaStream([
-          this.mediaProperties.audioTrack,
-          this.mediaProperties.videoTrack,
+          this.mediaProperties.audioTrack?.underlyingTrack,
+          this.mediaProperties.videoTrack?.underlyingTrack,
         ]),
       }
     );
   }
 
   /**
    * Sets the local audio track on the class and emits an event to the developer
-   * @param {MediaStreamTrack} audioTrack
+   * @param {MediaStreamTrack} rawAudioTrack
    * @param {Boolean} emitEvent if true, a media ready event is emitted to the developer
    * @returns {undefined}
    * @private
    * @memberof Meeting
    */
-  private setLocalAudioTrack(audioTrack: MediaStreamTrack, emitEvent = true) {
-    if (audioTrack) {
-      const settings = audioTrack.getSettings();
+  private setLocalAudioTrack(rawAudioTrack: MediaStreamTrack | null, emitEvent = true) {
```

Comment:
shouldnt setLocalAudioTrack take the LocalAudioTrack class rather then the rawAudioTrack 

is it because the web is passing the raw track i dont see a TODO

Reply:
Yes, this is temporary, because at the moment the public APIs for transcoded (addMedia, updateMedia, updateAudio, updateVideo, updateShare) take raw streams/tracks and also public APIs for multistream (publishTracks/unpublishTracks) take raw tracks. All APIs will be converted to use webrtc-core tracks in SPARK-399694 and SPARK-399695. I had to do this initial step without changing public APIs to fix the bug from [SPARK-407099](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-407099) quickly and without having to update the sample app and Cantina.